docs/plugins.md

```diff
@@ -290,145 +290,12 @@ exports.addWarning(message, options);
-    path: 'node_modules/protractor/plugins/ngHint',
-
-    asTests: {Boolean},
-    excludeURLs: {(String|RegExp)[]}
-  }]
-};
-```
-`asTests` specifies if the plugin should generate passed/failed test results
-based on the ngHint output or instead log the results to the console.
-Defaults to true.
-
-`excludeURLs` specifies a list of URLs for which ngHint results should be
-ignored. Defaults to []
-
-Timeline Plugin
----------------
-This plugin gathers test timeline information from the protractor test process, the selenium
-client logs (if available), and sauce labs (if available), and presents the output visually.
-This improves understanding of where latency issues are in tests.
-
-To enable the Timeline plugin, set it up in your config file:
-```js
-exports.config = {
-  plugins: [{
-   path: 'node_modules/protractor/plugins/timeline/index.js',
-
-    // Output json and html will go in this folder.
-   outdir: 'timelines',
-
-    // Optional - if sauceUser and sauceKey are specified, logs from
-   // SauceLabs will also be parsed after test invocation.
-     sauceUser: 'Jane',
-     sauceKey: 'abcdefg'
-   }],
-  // other configuration settings
-};
-```
-
-
-Console Plugin (Chrome only)
-----------------------------
-
-This plugin checks the browser log after each test for warnings and errors.  It
-can be configured to fail a test if either is detected.  There is also an
-optional exclude parameter which accepts both regex and strings.  Any log
-matching the exclude parameter will not fail the test or be logged to the
-console. A false setting to logWarnings also overrides the failOnWarning setting.
-
-```js
-exports.config = {
-  plugins: [{
-    path: 'node_modules/protractor/plugins/console',
-    failOnWarning: {Boolean}                (Default - false),
-    failOnError: {Boolean}                  (Default - true),
-    logWarnings: {Boolean}                  (Default - true),
-    exclude: {Array of strings and regex}   (Default - [])
-  }]
-};
-```
-
-Note that this plugin's behavior is undefined on browsers other than Chrome.
-Firefox users have reported flaky results.
-
-Community Plugins
------------------
+First Party Plugins
+-------------------
 
-This list is here for reference and the plugins included are not developed or mantained by protractor's team by any means. If you find any issues with this plugins please report them to the corresponding plugin developer.
+We maintain four plugins:
 
-* [Protractor testability plugin](https://github.com/alfonso-presa/protractor-testability-plugin): this plugins enables synchronous testing with protractor for features that are not developed using the services provided by AngularJS, preventing the need of additional waits coded in the tests. This happens for example if you have WebSockets communication with the server or for web applications built with frameworks different than AngularJS.
+* [Console Plugin (chrome only)](https://www.npmjs.com/package/protractor-console-plugin)
```

Comment:
Can we maintain the first-paragraph blurbs of what these plugins do? `Timeline Plugin` means very little without context.